javascript-data-model/order-history.js

```diff
@@ -0,0 +1,82 @@
+var orderHistory = [
+  {
+    orderDetails: {
+      orderPlacedDate: 'August 4, 2020',
+      totalPrice: '$34.00',
+      shipTo: 'JS Masher',
+      orderNumber: '114-3941689-8772232'
+    },
+    orderItems: [
+      {
+        deliveryDate: 'Aug 8, 2020',
+        itemImage: 'item-image.png',
+        itemName: 'JavaScript for impatient programmers',
+        bookAuthor: 'Rauschmayer, Dr. Axel',
+        returnWindow: 'Sep 7, 2020',
+        itemPrice: '$31.55'
+      }
+    ]
+  },
+  {
+    orderDetails: {
+      orderPlacedDate: 'July 19, 2020',
+      totalPrice: '$44.53',
+      shipTo: 'JS Masher',
+      orderNumber: '113-9984268-1280257'
+    },
+    orderItems: [
+      {
+        deliveryDate: 'Jul 20, 2020',
+        deliveryDetail: 'Your package was delivered. It was handed directly to a resident',
```

Comment:
You should have this property on each order for consistency, but `null` would be a good value for the orders that have no message

Reply:
This property would be better to have in the `orderDetails` object, since it is describing the order itself, not the individual item.